server/src/internalClusterTest/java/org/elasticsearch/recovery/RelocationIT.java

```diff
@@ -174,9 +248,12 @@ public void testRelocationWhileIndexingRandom() throws Exception {
             numberOfNodes
         );
 
+        // Start node with PAUSE_INDEXING_ON_THROTTLE setting set to true. This means that if we activate
+        // index throttling for a shard on this node, it will pause indexing for that shard until throttling
+        // is deactivated.
         String[] nodes = new String[numberOfNodes];
         logger.info("--> starting [node1] ...");
-        nodes[0] = internalCluster().startNode();
+        nodes[0] = internalCluster().startNode(Settings.builder().put(IndexingMemoryController.PAUSE_INDEXING_ON_THROTTLE.getKey(), true));
```

Comment:
Should we do this randomly? No need to not verify this for the regular throttling too.
```suggestion
        nodes[0] = internalCluster().startNode(Settings.builder().put(IndexingMemoryController.PAUSE_INDEXING_ON_THROTTLE.getKey(), randomBoolean()));
```

Reply:
Sure good idea. Took me a minute to parse "No need to not verify" 😄

Reply:
Sorry for my double negated danglish :slightly_smiling_face: